mldsa/src/sys.h

```diff
@@ -203,4 +211,37 @@
 #define MLD_MUST_CHECK_RETURN_VALUE
 #endif
 
+
+#if !defined(__ASSEMBLER__)
+/* System capability enumeration */
+typedef enum
+{
+#if defined(MLD_SYS_X86_64)
+  MLD_SYS_CAP_AVX2
+#elif defined(MLD_SYS_AARCH64)
+  MLD_SYS_CAP_SHA3
+#else
+  /* C90 does not allow empty enums, so use a dummy value
+   * for architectures other than AArch64 and x86_64. */
+  MLD_SYS_CAP_DUMMY
+#endif
```

Comment:
This breaks CBMC proofs on x86 - see https://github.com/pq-code-package/mlkem-native/issues/1318. 
Could you please change this to include my changes from https://github.com/pq-code-package/mlkem-native/pull/1319?